### PR TITLE
Zephyr integration changes

### PR DESCRIPTION
**Pull request by nashif**
- Reference local Kconfig to avoid inclusion of SOF Kconfig when SOF is not enabled from Zephyr
- Fix few formatting issues resulting in warning
- Revert reference to local copy of rimage which is now removed.

### CODE REVIEW

File: src/audio/kpb.c [hunk 1]
```diff
@@ -521,7 +521,7 @@ static int kpb_prepare(struct comp_dev *dev)
 	}
 
 	if (!kpb->sel_sink || !kpb->host_sink) {
-		comp_info(dev, "kpb_prepare(): could not find sinks: sel_sink %d host_sink %d",
+		comp_info(dev, "kpb_prepare(): could not find sinks: sel_sink %lu host_sink %lu",
```
Comment (lyakh) [inline src/audio/kpb.c:524]: those are actually pointers. This isn't a bug of this patch, but I don't understand why one would want to print pointers as (decimal) unsigned integers in the first place. So maybe we can use this opportunity and replace those with `%p` and remove type-casts below?

File: zephyr/CMakeLists.txt [hunk 2]
```diff
@@ -53,7 +53,7 @@ set(SOF_TRACE_PATH "${SOF_SRC_PATH}/trace")
 
 # Save path to rimage configuration files in cmake cache for later use by
 # rimage during the "west sign" stage
-get_filename_component(RIMAGE_CONFIG "ext/rimage/config" ABSOLUTE)
+get_filename_component(RIMAGE_CONFIG "../rimage/config" ABSOLUTE)
```
Comment (lyakh) [inline zephyr/CMakeLists.txt:56]: maybe we should remove this cached rimage path completely now and only use `-D`

**Review by lgirdwood — APPROVED** [review]
@nashif if you are busy then @lyakh has some bandwidth to fix up and merge ?

File: zephyr/CMakeLists.txt (hunk 2 above)
Comment (nashif) [reply zephyr/CMakeLists.txt:56]: yeah, we could

Comment (nashif) [reply zephyr/CMakeLists.txt:56]: on a second thought, I do not think there is a harm, this is just a variable being set. If we remove it now, we will have to also remove it from the signing script, we can do this later.

**Comment by nashif** [comment]
> @nashif if you are busy then @lyakh has some bandwidth to fix up and merge ?

addressed now.

**Review by lyakh — APPROVED** [review]
Thanks!

**Pull request merged**